tools/cli/internal/openapi/oasdiff.go

```diff
@@ -234,7 +234,9 @@ func (o OasDiff) mergeTags() error {
 			}
 		}
 	}
-	sort.Sort(ByName(baseTags))
+	slices.SortFunc(ByName(baseTags), func(a, b *openapi3.Tag) int {
+		return strings.Compare(strings.ToLower(a.Name), strings.ToLower(b.Name))
```

Comment:
why not https://pkg.go.dev/strings#EqualFold

Reply:
I will open a follow up PR, thanks!

Reply:
I see that equalFold returns a bool rather than an int 😭